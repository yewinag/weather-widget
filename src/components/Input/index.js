import styled from 'styled-components';

const Input = styled.input`
    font-size: 18px;
    padding: 10px;
    color: #000;
`;

export default Input;