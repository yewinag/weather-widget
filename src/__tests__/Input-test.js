import React from 'react';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

import Input from '../components/Input';

Enzyme.configure({ adapter: new Adapter() });

describe('input', () => {
    it('render input', () => {
        const wrapper = shallow(<Input />)
        expect(wrapper.exists()).toBe(true);
        expect(wrapper).toMatchSnapshot();
    });
});