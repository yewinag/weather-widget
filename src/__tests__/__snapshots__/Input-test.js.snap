// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input render input 1`] = `ShallowWrapper {}`;
